pkgs/development/gnuradio-modules/osmosdr/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv
+, mkDerivation
+, fetchgit
+, gnuradio
+, airspy
+, hackrf
+, libbladeRF
+, rtl-sdr
+, soapysdr-with-plugins
+}:
+
+let
+  version = {
+    "3.7" = "0.1.5";
+    "3.8" = "0.2.2";
+  }.${gnuradio.versionAttr.major};
+  src = fetchgit {
+    url = "git://git.osmocom.org/gr-osmosdr";
+    rev = "v${version}";
+    sha256 = {
+      "3.7" = "0bf9bnc1c3c4yqqqgmg3nhygj6rcfmyk6pybi27f7461d2cw1drv";
+      "3.8" = "HT6xlN6cJAnvF+s1g2I1uENhBJJizdADlLXeSD0rEqs=";
+    }.${gnuradio.versionAttr.major};
+  };
```

Comment:
Considering the complexity of `disabled` vs `enabledOnly`, I'm not even sure what would be the fate of these derivations when 3.9 will come..